--- src/orig.mainwindow_editor.cpp	2016-01-07 23:24:02.000000000 +0100
+++ src/mainwindow_editor.cpp	2016-01-13 20:00:43.000000000 +0100
@@ -61,7 +61,7 @@
     , m_widget(new QQuickWidget)
 {
     m_editorSession->setResourceManager(m_resourceManager);
-    setWindowIcon(QIcon::fromTheme("artikulate"));
+    setWindowIcon(QIcon::fromTheme("artikulate", windowIcon()));
     setWindowTitle(qAppName());
     setAutoSaveSettings();
 
